### ISSUE TITLE
Drive is not mapped after reboot

### ISSUE THREAD

**Issue by Zingam** [cyberponk/psubst#3]
On Windows 10 when I run the script for the first time. The drive was mapped and worked as expected. After rebooting thou the drive was not mapped. I checked the registry and the entry is there in `DOS Devices`.
Also running just `psubst` displays the mapped drive.

I used subst until yesterday but some MS tools do not work with drives mapped with subst.

BTW: The drive letter is reported as X:\\: which is not correct.

**Comment by cyberponk** [comment]
Could you tell me what version you are using  and what is the folder path
you are trying to subst?

**Comment by cyberponk** [comment]
And am example of a Ms tool that does not work so I can investigate

**Comment by Zingam** [comment]
Maybe it was a temporary Windows glitch. I reapplied `psubst` with `/F` and then I rebooted a few times. No problems any more.

I mean the MS tools work the way `psubst` maps a drive. If drives are mapped with the standard `subst` - there are issues. For example Visual Studio cannot deploy UWP apps from a `subst`-ed drive, nor do the packaging tool from the Xbox XDK.
I use `psubst` now (instead of mapping manually in the registry) and the tools work. The only issue is that the mappings are not per user. If there is a way to make them  per user it would be great.

Thank you for the great tool! I think it was a false alarm for now.
